src/loaders/IVolumeLoader.ts

```diff
@@ -179,10 +179,11 @@ export abstract class ThreadableVolumeLoader implements IVolumeLoader {
         const dtype = dtypes[i];
         const data = dataArrays[i];
         const range = ranges[i];
+        const time = volume.loadSpec.time;
```

Comment:
in VolumeDrawable you used loadSpecRequired.  make sure that's consistent and you're using the right one everywhere.  (I can't remember which is which, exactly)

Reply:
This was a really good question! I think from reviewing `Volume`, `loadSpecRequired` represents data that is required and _will_ be loaded, while `loadSpec` is data that _is_ loaded (or at least is currently in the process of being loaded).

Either will probably work but I think `loadSpec` is the right choice.

Reply:
Oops actually the changes in VolumeDrawable are a leftover from when time was in FuseChannel! Removing :>